packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -168,6 +169,7 @@ export class DatePicker extends BaseComponent<IDatePickerProps, IDatePickerState
       isRequired,
       disabled,
       ariaLabel,
+      pickerAriaLabel,
```

Comment:
Above you say `@defaultvalue Calendar` but there isn't a default value here or below where it's used.